packages/constants/src/surveys.ts

```diff
@@ -121,18 +132,24 @@ export const VITAL_CHARTS = {
 };
 
 // utility function for when a model's fields are all a direct match for their survey configs
-const makeLookupFields = (model: string, fields: string[]) =>
-  Object.fromEntries(fields.map(f => [f, [model, f]]));
+const makeLookupFields = (model: string, fields: (string | [string, Record<string, string>])[]) =>
+  Object.fromEntries(
+    fields.map(f => [Array.isArray(f) ? f[0] : f, [model, ...(Array.isArray(f) ? f : [f])]]),
+  );
 
 type PatientDataFieldLocationsType = {
-  [key: string]: Array<string>;
+  [key: string]: [string, string, Record<string, string>] | [string, string];
```

Comment:
preference would be to change this to some kind of descriptive interface, but probably out of scope; at least can you add a comment describing what the different string fields of the array do please